### DISCUSSION TITLE
MySQL healthcheck configuration

### DISCUSSION THREAD

**Discussion by alitheg** [Security-Onion-Solutions/securityonion#2599] [Unsupported Versions]
I noticed my `so-status` is showing the MySQL container as "unhealthy". I've logged into the container to inspect it, and it looks like `healthcheck.sh` is looking for files called `/mysql-init-complete` and `/healthcheck.cnf`, which are creaetd by the entrypoint script, but it looks like the entrypoint script actually creates these in /tmp instead, so the healthcheck fails to find them and reports unhealthy. I think this is a bug - can anyone else see the same issue?

**Comment by jertel** [comment]
Thanks for reporting this. I've created issue #2605 for tracking it.
